src/course/en/components.json

```diff
@@ -572,11 +563,15 @@
 		"_scaleEnd": 10,
 		"title": "Slider",
 		"displayTitle": "Slider",
-		"body": "Working memory is thought to be responsible for our ability to temporarily hold and manipulating information but according to a paper by Miller (1956) what ‘magic number’ describes its capacity?",
+		"body": "Working memory is thought to be responsible for our ability to temporarily hold and manipulate information, but according to a paper by Miller (1956) what ‘magic number’ describes its capacity?",
 		"instruction": "Drag the slider to make your choice and select Submit.",
 		"labelStart": "",
 		"labelEnd": "",
-		"_correctAnswer": "7",
+		"_correctAnswer": "",
+		"_correctRange": {
+			"_bottom": 7,

```

Comment:
shouldn't this be 5? Feedback says 'According to Miller’s paper 7 +/- 2 chunks of information was the limited capacity of working memory'